src/plugins/vis_type_xy/public/types/config.ts

```diff
@@ -54,6 +54,8 @@ export interface Aspects {
   y: Aspect[];
   z?: Aspect;
   series?: Aspect[];
+  splitColumn?: Aspect;
```

Comment:
@nickofthyme  if I'm not wrong we cannot make split by column + split by name. 
Maybe instead of that  2 props we can do something like: 

```
enum SplitBy {
  Row = 'row',
  Column = 'column',
}

export interface Aspects {
  ...
  splitBy?: SplitBy
  ...
```